DigitalLearningSolutions.Data/Services/ActionPlanService.cs

```diff
@@ -148,8 +152,8 @@ public async Task AddResourceToActionPlan(int learningResourceReferenceId, int d
                 );
 
             return response.resource != null
-                ? new ActionPlanResource(learningLogItem, response.resource)
-                : null;
+                ? (new ActionPlanResource(learningLogItem, response.resource), response.sourcedFromFallbackData)
+                : (null, false);

```

Comment:
Let's not hardcode `false` here - let's just pass the value through from GetLearningLogItems() (see above comment for motivation).

Reply:
Should be changed anywhere else where it's applicable too.

Reply:
Ah right, I've passed apiIsAccessible through rather than hard coding